docs/reference/ingest/apis/enrich/delete-enrich-policy.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[delete-enrich-policy-api]]
+=== Delete enrich policy API
+++++
+<titleabbrev>Delete enrich policy</titleabbrev>
+++++
+
+Deletes an existing enrich policy and its enrich index.
+
+////
+[source,js]
+----
+PUT /users
+
+PUT /_enrich/policy/my-policy
+{
+    "match": {
+        "indices": "users",
+        "match_field": "email",
+        "enrich_fields": ["first_name", "last_name", "city", "zip", "state"]
+    }
+}
+----
+// CONSOLE
+// TESTSETUP
+////
+
+[source,js]
+--------------------------------------------------
+DELETE /_enrich/policy/my-policy
+--------------------------------------------------
+// CONSOLE
+
+
+[[delete-enrich-policy-api-request]]
+==== {api-request-title}
+
+`DELETE /_enrich/policy/<enrich-policy>`
+
+
+[[delete-enrich-policy-api-prereqs]]
+==== {api-prereq-title}
+
+include::put-enrich-policy.asciidoc[tag=enrich-policy-api-prereqs]
+
+
+[[delete-enrich-policy-api-desc]]
+==== {api-description-title}
+
+Use the delete enrich policy API
+to delete an existing enrich policy
+and its enrich index.
+
+[IMPORTANT]
+====
+You must remove an enrich policy
+from any in-use enrich processors and ingest pipelines
```

Comment:
maybe change `enrich processors and ingest pipelines` into `ingest pipeline`?
Processors are part of a pipeline's configuration, so the latter is less redundant.

Reply:
Fixed with 0f17b487f90fba0677aba9f4240ecdb8f33d7777.